docs/src/pages/en/guides/rss.md

```diff
@@ -41,3 +43,11 @@ export async function getStaticPaths({rss}) {
 ```
 
 Note: RSS feeds will **not** be built during development. Currently, RSS feeds are only generated during your final build.
+
+### Styling
+
+RSS Feeds can be styled with an XSL stylesheet for a more pleasant user experience when they are opened directly in a browser. By default, Astro does not set a stylesheet for RSS feeds, but it can be enabled by setting the `stylesheet` option.
```

Comment:
Mind adding a link to https://developer.mozilla.org/en-US/docs/Web/XSLT / something like https://natclark.com/tutorials/xslt-style-rss-feed/ ?